tracing-core/src/lib.rs

```diff
@@ -219,6 +216,14 @@ pub mod span;
 pub(crate) mod stdlib;
 pub mod subscriber;
 
+// Vendored version of spin 0.5.2 (0387621)
+// `mutex` and `once` modules only
+#[cfg(not(feature = "std"))]
+pub mod spin;
+
+// Vendored version of lazy_static 1.4.0 (4216696)
```

Comment:
Similarly, it would be nice to add a note explaining the motivation for vendoring.